src/data/text/move_names.h

```diff
@@ -1,3 +1,766 @@
+    [MOVE_BANEFUL_BUNKER] = _("Baneful Bunker"),
+    [MOVE_SPIRIT_SHACKLE] = _("Spirit Shackle"),
+    [MOVE_DARKEST_LARIAT] = _("Darkest Lariat"),
+    [MOVE_SPARKLING_ARIA] = _("Sparkling Aria"),
+    [MOVE_ICE_HAMMER] = _("Ice Hammer"),
+    [MOVE_FLORAL_HEALING] = _("Floral Healng"),
+    [MOVE_HIGH_HORSEPOWER] = _("High Horsepower"),
+    [MOVE_STRENGTH_SAP] = _("Strength Sap"),
+    [MOVE_SOLAR_BLADE] = _("Solar Blade"),
+    [MOVE_LEAFAGE] = _("Leafage"),
+    [MOVE_SPOTLIGHT] = _("Spotlight"),
+    [MOVE_TOXIC_THREAD] = _("Toxic Thread"),
+    [MOVE_LASER_FOCUS] = _("Laser Focus"),
+    [MOVE_GEAR_UP] = _("Gear Up"),
+    [MOVE_THROAT_CHOP] = _("Throat Chop"),
+    [MOVE_POLLEN_PUFF] = _("Pollen Puff"),
+    [MOVE_ANCHOR_SHOT] = _("Anchor Shot"),
+    [MOVE_PSYCHIC_TERRAIN] = _("Psychic Terrain"),
+    [MOVE_LUNGE] = _("Lunge"),
+    [MOVE_FIRE_LASH] = _("Fire Lash"),
+    [MOVE_POWER_TRIP] = _("Power Trip"),
+    [MOVE_BURN_UP] = _("Burn Up"),
+    [MOVE_SPEED_SWAP] = _("Speed Swap"),
+    [MOVE_SMART_STRIKE] = _("Smart Strike"),
+    [MOVE_PURIFY] = _("Purify"),
+    [MOVE_REVELATION_DANCE] = _("Revelation Dance"),
+    [MOVE_CORE_ENFORCER] = _("Core Enforcer"),
+    [MOVE_TROP_KICK] = _("Trop Kick"),
+    [MOVE_INSTRUCT] = _("Instruct"),
+    [MOVE_BEAK_BLAST] = _("Beak Blast"),
+    [MOVE_CLANGING_SCALES] = _("Clanging Scales"),
+    [MOVE_DRAGON_HAMMER] = _("Dragon Hammer"),
+    [MOVE_BRUTAL_SWING] = _("Brutal Swing"),
+    [MOVE_AURORA_VEIL] = _("Aurora Veil"),
+    [MOVE_SHELL_TRAP] = _("Shell Trap"),
+    [MOVE_FLEUR_CANNON] = _("Fleur Cannon"),
+    [MOVE_PSYCHIC_FANGS] = _("Psychic Fangs"),
+    [MOVE_STOMPING_TANTRUM] = _("Stomping Tantrum"),
+    [MOVE_SHADOW_BONE] = _("Shadow Bone"),
+    [MOVE_ACCELEROCK] = _("Accelerock"),
+    [MOVE_LIQUIDATION] = _("Liquidation"),
+    [MOVE_PRISMATIC_LASER] = _("Prismatic Laser"),
+    [MOVE_SPECTRAL_THIEF] = _("Spectral Thief"),
+    [MOVE_SUNSTEEL_STRIKE] = _("Sunsteel Strike"),
+    [MOVE_MOONGEIST_BEAM] = _("Moongeist Beam"),
+    [MOVE_TEARFUL_LOOK] = _("Tearful Look"),
+    [MOVE_ZING_ZAP] = _("Zing Zap"),
+    [MOVE_NATURES_MADNESS] = _("Nature's Madness"),
+    [MOVE_MULTI_ATTACK] = _("Multi-Attack"),
+    [MOVE_MIND_BLOWN] = _("Mind Blown"),
+    [MOVE_PLASMA_FISTS] = _("Plasma Fists"),
+    [MOVE_PHOTON_GEYSER] = _("Photon Geyser"),
+    [MOVE_ZIPPY_ZAP] = _("Zippy Zap"),
+    [MOVE_SPLISHY_SPLASH] = _("Splishy Splash"),
+    [MOVE_FLOATY_FALL] = _("Floaty Fall"),
+    [MOVE_PIKA_PAPOW] = _("Pika Papow"),
+    [MOVE_BOUNCY_BUBBLE] = _("Bouncy Bubble"),
+    [MOVE_BUZZY_BUZZ] = _("Buzzy Buzz"),
+    [MOVE_SIZZLY_SLIDE] = _("Sizzly Slide"),
+    [MOVE_GLITZY_GLOW] = _("Glitzy Glow"),
+    [MOVE_BADDY_BAD] = _("Baddy Bad"),
+    [MOVE_SAPPY_SEED] = _("Sappy Seed"),
+    [MOVE_FREEZY_FROST] = _("Freezy Frost"),
+    [MOVE_SPARKLY_SWIRL] = _("Sparkly Swirl"),
+    [MOVE_VEEVEE_VOLLEY] = _("Veevee Volley"),
+    [MOVE_DOUBLE_IRON_BASH] = _("Double Iron Bash"),
+    //GEN 8
```

Comment:
```suggestion
    // GEN 8
```
Though I don't know why is this comment here at all. There's no point to it, unless we added one to separate each generation's set of moves.